/usr/bin/kvm doesn't exist on suse distros so check instead for
/usr/bin/qemu-system-x86_64
Index: virt-manager-5.1.0/virtManager/createconn.py
===================================================================
--- virt-manager-5.1.0.orig/virtManager/createconn.py
+++ virt-manager-5.1.0/virtManager/createconn.py
@@ -27,7 +27,7 @@ def _default_uri():  # pragma: no cover
     if (
         os.path.exists("/usr/bin/qemu")
         or os.path.exists("/usr/bin/qemu-kvm")
-        or os.path.exists("/usr/bin/kvm")
+        or os.path.exists("/usr/bin/qemu-system-x86_64")
         or os.path.exists("/usr/libexec/qemu-kvm")
         or glob.glob("/usr/bin/qemu-system-*")
     ):
